cylc/flow/scheduler.py

```diff
@@ -1269,6 +1269,8 @@ def _configure_contact(self) -> None:
     def load_flow_file(self, is_reload=False):
         """Load, and log the workflow definition."""
         # Local workflow environment set therein.
+        self.options.rose_template_vars = []
```

Comment:
Please add a comment, else we will forget why we do this.

Reply:
```suggestion
        # https://github.com/cylc/cylc-flow/issues/5968
        self.options.rose_template_vars = []
```